spec/index.md

```diff
@@ -193,50 +198,18 @@ This follows [Data Package][dp] (DP). In particular, the following properties `M
 ```
 
 
-
-## Resources
```

Comment:
I removed it because I moved the actual content to [here](https://github.com/openspending/fiscal-data-package/pull/92/files#diff-213427d3e24797581154795a8bb3ab22L174).

Basically, I'm trying to get rid of as much text discussing the JSON structure as possible, in favour of succinct comments in the JSON itself.

An example of a resource within the JSON would be a good thing - and for that matter, making the whole JSON file complete, and the basis of all the examples.